stories/CircularProgressBar.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { CircularProgressBar } from "../src/components/CircularProgressBar";
+
+const meta = {
+    title: "Components/CircularProgressBar",
+    component: CircularProgressBar,
+    parameters: {
+        layout: "centered",
+    },
```

Comment:
```suggestion
    },
        tags: ["autodocs"],
```
this will generate the documentation page

Reply:
Thanks I will add this.